apps/backend/src/routes/me/index.ts

```diff
@@ -46,4 +49,60 @@ router.post<UserUpdateRequestBody, ApiResponseBody<UserUpdateResponseBody>>(
   }
 );
 
+router.post<TcxFileUpload, ApiResponseBody<StravaUpload>>(
```

Comment:
Why are we doing this on the backend? Isn't it better to do this on the frontend? Just make the frontend request the access token for Strava and do the post directly?

Reply:
Think we should look into just doing it client side later, but I think we should just ship this 🚀